docs/diagnostics/PX1007.md

```diff
@@ -8,7 +8,10 @@ This document describes the PX1007 diagnostic.
 | PX1007 | A DAC, a DAC extension, and public DAC properties should have a description in the `summary` XML tag. | Warning | Available | 
 
 ## Diagnostic Description
-Any DAC or DAC extension and their public property fields should have a description in the `summary` XML tag. The descriptions of the DACs, DAC extensions, and their members can be displayed in the DAC Schema Browser and used for building the API and DAC references for the Acumatica ERP libraries.
+Any DAC or DAC extension and their public property fields should have one of the following:
```

Comment:
There should be many more details, the diagnostic became rather complex. You can find them here:
https://github.com/Acumatica/Acuminator/pull/497#issue-1722660403

The documentation should mention that PX1007 now supports one or multiple inheritdoc tags in any format on DACs, DAC extensions and DAC field properties of non projection DACs (and even unmapped properties of projection DACs). 
But it has strict rules regarding the mapped properties of projection DACs.